pkg/k8s/volumes/crud.go

```diff
@@ -47,29 +47,29 @@ func List(ctx context.Context, namespace, labels string, c kubernetes.Interface)
 }
 
 // CreateForDev deploys the volume claim for a given development container
-func CreateForDev(ctx context.Context, dev *model.Dev, c *kubernetes.Clientset, devPath string) error {
+func CreateForDev(ctx context.Context, dev *model.Dev, c kubernetes.Interface, devPath string) error {
 	vClient := c.CoreV1().PersistentVolumeClaims(dev.Namespace)
-	pvc := translate(dev)
-	k8Volume, err := vClient.Get(ctx, pvc.Name, metav1.GetOptions{})
+	pvcFromDev := translate(dev)
+	k8Volume, err := vClient.Get(ctx, pvcFromDev.Name, metav1.GetOptions{})
 	if err != nil && !strings.Contains(err.Error(), "not found") {
 		return fmt.Errorf("error getting kubernetes volume claim: %s", err)
 	}
 	if k8Volume.Name == "" {
-		oktetoLog.Infof("creating volume claim '%s'", pvc.Name)
-		_, err = vClient.Create(ctx, pvc, metav1.CreateOptions{})
+		oktetoLog.Infof("creating volume claim '%s'", pvcFromDev.Name)
+		_, err = vClient.Create(ctx, pvcFromDev, metav1.CreateOptions{})
 		if err != nil {
 			return fmt.Errorf("error creating kubernetes volume claim: %s", err)
 		}
 	} else {
-		if err := checkPVCValues(pvc, dev, devPath); err != nil {
+		if err := checkPVCValues(k8Volume, dev, devPath); err != nil {
```

Comment:
Since when have been like that? I know this is the expected and correct behavior but I guess this can start failing where it wasn't before and it might increase the friction.

Can we include the possible impact? I mean, with the current code, what was happening? what was wrong? Was everything working fine or this was causing an error later?

Which implies the new change? Can fail scenarios that were working before?

Reply:
I can't think of any scenario that was working (without any error on up) to start failing with these change. The only thing that is done is checking if we are going to have an error before we get the k8s error

Reply:
@ifbyol the change was originated [here](https://github.com/okteto/okteto/pull/1586/files). It seems to be a bug since before it was done as the proposed fix.

Reply:
@ifbyol I've added more description to the PR to explain the impact of the changes

Reply:
ok, thanks for the information. I think we can merge it and no working scenarios should be affected